app/models/Project.js

```diff
@@ -0,0 +1,102 @@
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema
+const validator = require('validator')
+
+const projectSchema = new Schema({
+  projectName: {
+    type: String,
+    required: true,
+    trim: true,
+    minlength: 3,
+    validate (projectName) {
+      if (validator.isEmpty(projectName)) {
+        throw new Error('Project name is required!')
+      }
+      if (!validator.isLength(projectName, { min: 3 })) {
+        throw new Error('Project name should be 3 characters long!')
+      }
+    }
+  },
+  description: {
+    short: {
+      type: String,
+      required: true,
+      trim: true,
+      minlength: 10,
+      validate (short) {
+        if (validator.isEmpty(short)) {
+          throw new Error('Short description for the project is required!')
+        }
+        if (!validator.isLength(short)) {
+          throw new Error('Short description should be min 10 characters long!')
+        }
+      }
+    },
+    long: {
+      type: String,
+      trim: true
+    }
+  },
+  image: {
```

Comment:
in one project there can be multiple images, how are we handling that?

Reply:
its okay for now we can take this up at later point in time

Reply:
oh, then we should create an array of imageIds, I think aws create unique ids for each image?
and during rendering we can just fetch the image according to the ids.
What do you think?